Extend the `Location.assign` page with other URL options (#31006)
mdn/content: files/en-us/web/api/location/assign/index.md

### Parameters

- `url`

>   - : A string containing the URL of the page to navigate to. It can be either an absolute URL or a relative URL.

Reviewer: ```suggestion
  - : A string containing the URL of the page to navigate to; for example, an absolute URL such as `https://developer.mozilla.org/en-US/docs/Web/API/Location/reload`, or a relative URL — such as `"/Web` (just a path, for navigating to another document at the same origin) or `#specifications` (just a fragment string, for navigating to some part of the same page), and so on.
```

>   - : A string containing the URL of the page to navigate to. It can be an absolute and full path, as well as relative path to a subdirectory of website.

Reviewer: ```suggestion
  - : A string containing the URL of the page to navigate to. It can be either an absolute URL or a relative URL.
```

window.location.assign(
  "https://developer.mozilla.org/en-US/docs/Web/API/Location/reload",
);

> // Then navigate to it's #Specifications section

Reviewer: ```suggestion
// Then navigate to its Specifications section
```

> window.location.assign("#specification");

Reviewer: ```suggestion
window.location.assign("#specifications");
```
